src/PythonQtImporter.cpp

```diff
@@ -308,7 +328,11 @@ PythonQtImporter_load_module(PyObject *obj, PyObject *args)
     }
 
     Py_DECREF(code);
+#if PY_VERSION_HEX >= 0x030C0000  // Python >= 3.12
+    if (getSysFlag("verbose")) {
```

Comment:
Probably makes sense to add convenience functions for this often used flag (and maybe other flags), instead of cluttering the code with `#ifdef`s.